gvsbuild/projects/__init__.py

```diff
@@ -81,4 +81,5 @@
 from gvsbuild.projects.win_iconv import WinIconv
 from gvsbuild.projects.wing import Wing
 from gvsbuild.projects.x264 import X264
+from gvsbuild.projects.xml import Expat, Libxml2
```

Comment:
Why this? we are keeping a file-per-project for everything else, I think we should keep the same here as well

Reply:
I was thinking that it might be clear to bundle xml related stuff in the same module. You are right, I reverted the commit.